definitions/windows_bootstrap.yaml

```diff
@@ -44,8 +102,8 @@ provides: [environ_programfiles]
 supported_os: [Windows]
 urls: ['http://environmentvariables.org/ProgramFiles']
 ---
-name: ProgramFilesx86
-doc: The %ProgramFiles (x86)% environment variable.
+name: WindowsEnvironmentVariableProgramFilesX86
```

Comment:
I think we need to pick a consistent naming scheme. Either everything Windows starts with Windows, or nothing does.